app/services/vot/acr_component_values.rb

```diff
@@ -98,8 +125,22 @@ module AcrComponentValues
       [component_value.name, component_value]
     end.to_h.freeze
 
+    DELIM = ' '
+
     def self.by_name
       NAME_HASH
     end
+
+    # @param acr_values [String,Array<String>]
+    def self.any_semantic_acrs?(acr_values)
+      return false unless acr_values.present?
+      # @type [Array]
+      values = (
+                 acr_values.is_a?(String) && acr_values.presence.split(DELIM) ||
```

Comment:
[suggestion] you can get rid of the `presence` check

Reply:
done!